inputQueueSize: 1
tfMapFrame: /map

mapSamplingDensity: 20

usetoyproblem: false

#Select Matcher (only one)
usetemplate: false
useGoICP: true
useSuper4PCS: false
usepclPlaneExtraction: false
useRHTPlaneExtraction: false

